src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx

```diff
@@ -47,12 +52,89 @@ const ReviewChangeRequestsView: React.FC<ReviewChangeRequestViewProps> = ({
     reset({ reviewNotes: '' });
   };
 
+  const overflow: object = {
+    'overflow-y': 'scroll',
+    'max-height': '300px'
+  };
+
+  const proposedSolutionStyle = {
+    cursor: 'pointer',
+    width: 'auto',
+    margin: 'auto',
+    display: 'block'
+  };
+
+  if (cr.type === 'ISSUE') {
+    const issueCR = cr as StandardChangeRequest;
+    return (
+      <Modal
```

Comment:
since both these modals are so long can you make a renderModalProposedSolutions and a renderModal function in this file and then call them where appropriate? These would go below your style object but above the ternary operator I specified above. See https://github.com/Northeastern-Electric-Racing/FinishLine/pull/182 TeamSummary.tsx if you wanna see an example (renderTeamSummaryHeader ). It's just a nice way of making the code more readable when the files get long like this

Reply:
Done. Left as an if-else though since ternary looked a big ugly with multiple conditionals through an OR.